content/en/os/1.14.x/concepts/bootstrap-containers/_index.markdown

```diff
@@ -0,0 +1,88 @@
++++
+title = "Bootstrap Containers"
+type = "docs"
+description = "Setting up the host with containers that start during boot." 
++++
+
+Bootstrap containers are a form of {{< ver-ref project="os" page="/concepts/components" >}}host container{{< /ver-ref >}} that you can use to run critical software before the node connects to an orchestrator.
+They give you subsantial power to configure and modify the system in ways that would otherwise be [difficult or impossible](#use-cases).
+
+## Lifecycle
+
+When you define a bootstrap container using user-data or the Bottlerocket API, the following sequence occurs on next boot:
+
+- [`systemd`](https://systemd.io/) starts all of the bootstrap containers concurrently,
+- the boot process pauses while the bootstrap containers start, run, and exit,
```

Comment:
This isn't completely accurate. What I read from this is that systemd won't run any other service until all bootstrap containers complete. Systemd will run many services in parallel, but it won't move to the next target (that is `multi-user.target`) until all the bootstrap containers succeed. So technically, the boot process doesn't completely stop, since as I said, more work will be done by systemd if it can.